.github/workflows/cropmask-test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Check files using the black formatter
       uses: lgeiger/black-action@master
       with:
-        args: "production/dea_ml --check"
```

Comment:
Removing this will mean that this action changes the files, which it can't when running in CI.

Reply:
add it back